Gemfile

```diff
@@ -73,11 +73,14 @@ gem "front_matter_parser", "~> 1.0"
 gem "kramdown", "~> 2.3"
 gem "rails-reverse-proxy", "~> 0.11"
 gem "jwt", "~> 2.3"
+gem "rss", "~> 0.2"
+gem "faraday", "~> 2.3"
 
 group :test do
   gem "climate_control", "~> 1.0"
   gem "simplecov", "~> 0.21", require: false
   gem "shoulda-matchers", "~> 5.1"
   gem "rails-controller-testing", "~> 1.0"
   gem "webmock", "~> 3.14"
+  gem "vcr", "~> 6.1"
```

Comment:
VCR is a testing gem that records network calls (in this case, a request for the blog RSS) and replays it back the next time the test is run. This lets us not worry about mocking out every aspect of a valid RSS feed while also not having to make network requests every single test run.

Reply:
Very neat 👍